fxsampler/src/main/java/fxsampler/FXSampler.java

```diff
@@ -367,31 +348,51 @@ private String formatSourceCode(Sample sample) {
         String template = getResource("/fxsampler/util/SourceCodeTemplate.html", null);
         return template.replace("<source/>", src);
     }
-    
-    
+
+    private String getSourceCode(Sample sample) {
+        String sourceURL = sample.getSampleSourceURL();
+        try {
+            // try loading via the web or local file system
+            URL url = new URL(sourceURL);
+            InputStream is = url.openStream();
+            return getResource(is);
+        } catch (IOException e) {
+            // no-op - the URL may not be valid, no biggy
+        }
+        return getResource(sourceURL, sample.getClass());
+    }
+
     private String formatCss(Sample sample) {
         String cssUrl = sample.getControlStylesheetURL();
         String src;
         if (cssUrl == null) {
             src = "No CSS source available";
         } else {
-	        src = "Css not found";
-	        try {
-	        	src = new String(
-					Files.readAllBytes( Paths.get(getClass().getResource(cssUrl).toURI()) )
-				);
-	        } catch(Throwable ex){
-	            ex.printStackTrace();
-	        }
+            src = "Css not found";
+            try {
+                final Optional<Module> fxsamplerModule = ModuleLayer.boot().findModule("org.controlsfx.fxsampler");
+                if (fxsamplerModule.isPresent() && selectedProject != null) {
+                    // module-path
+                    final Optional<Module> controlsFXModule = ModuleLayer.boot().findModule(selectedProject.getModuleName());
```

Comment:
this has nothing to do with controlsFX, has it? Rename to `projectModule` or alike?